example/package.json

```diff
@@ -36,6 +36,7 @@
     "@cumulus/deployment": "^1.11.3",
     "@cumulus/discover-granules": "^1.11.3",
     "@cumulus/discover-pdrs": "^1.11.3",
+    "@cumulus/files-to-granules": "^1.11.3-alpha1",
```

Comment:
Remember to remove this `-alpha1` before merging.

Reply:
I'm pretty sure I need to publish it before merging to master. There is no `@cumulus/files-to-granules` on npm.